recipe/meta.yaml

```diff
@@ -42,6 +43,9 @@ test:
     - python 3.5.*  # [win and py35]
 
   commands:
+    # Inspect linkages of HDF5.
+    - conda inspect linkages -n _test hdf5  # [unix]
```

Comment:
Unfortunately, this fails due to this issue ( https://github.com/conda/conda-build/issues/836 ).